content/en/platform/corda/4.8/open-source/reissuing-states.md

```diff
@@ -34,7 +34,7 @@ The new state reissuance functionality provides a state reissuance algorithm tha
 {{< note >}}
 State encumbrance refers to a state pointing to another state that must also appear as an input to any transaction consuming this state. A state may be encumbered by up to one other state, which is called an "encumbrance" state. The encumbrance state, if present, forces additional controls over the encumbered state, since the encumbrance state contract will also be verified during the execution of the transaction.
 
-See [Defining encumbrances](../../../../tutorials/corda/4.8/os/build-basic-cordapp/basic-cordapp-contract.md) for more information.
+See [Encumbrances](../../../../en/platform/corda/4.8/open-source/key-concepts-contracts.html#encumbrances) for more information.
```

Comment:
NB this link fix is something I worked on recently. It's not in the scope of this ticket, but I saw it while I was here (and checked in other versions, this is the only version that needed to be fixed).